api/proto/config/edgeview.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright(c) 2021 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+package org.lfedge.eve.config;
+option go_package  = "github.com/lf-edge/eve/api/go/config";
+option java_package = "org.lfedge.eve.config";
+
+message EdgeViewConfig {
+    // JWT token for signed info, it contains the dispatcher
+    // endpoint IP:Port, device UUID and expiration time
+    string  token = 1;
+    // dispatcher certificate(s) if it's not well-known CA signed
+    repeated bytes disp_cert_pem = 2;
+    // policy for device access through edge-view
+    DevEdgeViewPolicy dev_policy = 3;
+    // policy for apps through edge-view
+    AppEdgeViewPolicy app_policy = 4;
+}
+
+message DevEdgeViewPolicy {
+    // device side of edge-view access is alllowed or not
+    bool allow_dev = 1;
+    // allow ssh into device or not, ssh key still need to be
+    // installed separately even the edge-view allows it
+    bool ssh = 2;
+    // allow to issue reboot device or not
+    bool reboot = 3;
```

Comment:
Is this specifically via edge-view? If so it needs to be named with that in mind.

Reply:
Yes, as above mentioned, this policy is only for edge-view. Should it be 'allow_shell_reboot' ?